spot-client/src/common/css/page-layouts/volume-modal.scss

```diff
@@ -0,0 +1,18 @@
+@import './../variables.scss';
+
+.volume-modal {
+    align-items: center;
+    background-color: var(--container-bg-color);
+    display: flex;
+    flex: 1;
+    flex-direction: column;
+    padding: 50px;
+
+    span {
+        background-color: rgba(0, 0, 0, 0.5);
```

Comment:
I try, and probably fail, at keeping colors as CSS variables. Sometimes I give up because there are so many one-off colors. I want CSS variables because that way in the future we can do custom color theming using JS, although that plan might have some bumps because Yana had mentioned once using material-UI and using its theming ability.

Reply:
Good point, I move them to the variables too. Thanks